app/services/duty_options/base.rb

```diff
@@ -1,6 +1,9 @@
 module DutyOptions
   class Base
     include ActionView::Helpers::NumberHelper
+    include ServiceHelper
+
+    COUNTERVAILING_ANTI_DUMPING_MEASURE_TYPE_IDS = %w[551 552 553 554].freeze
```

Comment:
How do you feel about moving this knowledge to the measure type and exposing it behind a method?

```ruby
REMEDIAL_IDS = %w[551 552 553 554].freeze

def remedial?
  id.in?(REMEDIAL_IDS)
end

# In DutyOptions::Base

def localized_footnote
  return I18n.t("measure_type_footnotes.#{measure_type_id}", link: link) if measure.measure_type.remedial?

  I18n.t("measure_type_footnotes.#{measure.measure_type.id}") 
end
```

Reply:
You've always got the measure in scope of a duty option (since they are highly related)

Reply:
yep sounds good!